packages/frontend/src/i18n/resources/en.ts

```diff
@@ -5,6 +5,9 @@ export const en = {
         "connect.wallet": "Connect wallet",
         "connect.wallet.network": "Network",
         "home.createKPIToken": "Create KPI token",
+        "home.noFeatured.title": "Reach your goals with a Carrot",
+        "home.noFeatured.description":
+            "Easy and powerful tool to create conditional tokens allowing an effective method to achieve any goal desirable using permissionless, decentralized technologies.",
```

Comment:
I think we should shorten this message, it's too much text to read.

Something like 

> Easy and powerful tool to create conditional tokens to achieve any goal.

Reply:
It's a description... the title is shorter, the description goes into more detail. I think we can leave it like this, we'll probably discuss the "proper" text content later anyway.